Parity Check Tuning/paritychecktuning.txt

```diff
@@ -20,9 +20,9 @@ Resume an array operation at=
 Send notifications for temperature related Pause or Resume=
 Parity Check Tuning debug logging=
 Requires Unraid 67 or later=Requires Unraid 6.7 or late
-ParityRead-Check History=Parity/Read-Check History=
-Click to show parity-check history=
-unknown action %s=unknown action: %s
+ParityRead-Check History=Parity/Read-Check History=Bisherige Parity-Prüfungen
```

Comment:
Bisherige Paritäts Prüfungen
@spencerjunraid is in this line something wrong? I think the english text is doubled?